src/p11_pkey.c

```diff
@@ -300,9 +300,9 @@ static int pkcs11_params_oaep(CK_RSA_PKCS_OAEP_PARAMS *oaep,
 	oaep->mgf = pkcs11_md2ckg(mgf1_md);
 	if (!oaep->hashAlg || !oaep->mgf)
 		return -1;
+	oaep->source = 1UL; /* CKZ_DATA_SPECIFIED */
```

Comment:
Please, use the constant directly -- its defined in the `pkcs11.h`. Using magic numbers leads only to errors like this.

https://github.com/OpenSC/libp11/blob/5e25cc823e6d569bcb35203b2b2c595086adf7e4/src/pkcs11.h#L869

Reply:
Done — force pushed

Reply:
Done — force pushed